geo_autoRIFT/autoRIFT/autoRIFT.py

```diff
@@ -54,13 +58,13 @@ def _wallis_filter(image, filter_width):
     kernel = kernel / np.sum(kernel)
 
     shifted = _remove_local_mean(image, kernel)
-    std = _preprocess_filt_std(shifted, kernel)
-
+    std = _preprocess_filt_std(image, kernel)
+    std[std == 0] = np.nan
```

Comment:
Since `std` is a `float`, I think it probably makes sense to use [`np.isclose`](https://numpy.org/doc/stable/reference/generated/numpy.isclose.html) here